include/updater/csv_file_parser.hpp

```diff
@@ -122,7 +122,7 @@ template <typename Key, typename Value> struct CSVFilesParser
 
             util::Log() << "Loaded " << filename << " with " << result.size() << "values";
 
-            return std::move(result);
+            return result;
```

Comment:
result structure can be huge and it will be better to avoid copying.
Will it fix errors if you will add a std:move statement to the previous return at line 99?

Reply:
I've removed these calls to std::move as the compiler throws an error saying the call to std::move is less efficient then letting the compiler optimize this move away completely with [copy elision](https://en.cppreference.com/w/cpp/language/copy_elision).

The other way to address this would be to turn off compiler warnings for the 'pessimistic move' error and keep the moves, I prefer to remove std::move calls, but I can have a look at the compiler warning options if that is preferable?

Reply:
Thank you. I understand what's happened. I think we can leave it this way (without move) if there is no other occurrences of this warning.

Can you add this compiler to the travis config?